fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJobRunner.java

```diff
@@ -205,17 +204,25 @@ private void executeInsert() throws Exception {
 
     private void createTempPartitions() throws DdlException {
         long createPartitionStartTimestamp = System.currentTimeMillis();
+        Database db = getAndWriteLockDatabase(dbId);
```

Comment:
You get db write lock here but not release it.

Reply:
fixed